### PR TITLE
chore(ocean-api): initial nestjs monorepo setup

### PR DESCRIPTION
**Pull request by kodemon**
#### What this PR does / why we need it:

A pre-cursor to workflow and shift left adjustments in jellyfish to prepare for consolidation with playground and whale. Also preps for deployment tickets #942 #943

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,7 +23,12 @@
     "all:clean": "rm -rf ./packages/**/dist && rm -rf ./apps/**/dist && rm -rf ./packages/**/tsconfig.build.tsbuildinfo",
     "all:build": "lerna run build",
     "all:version": "lerna version $1 --yes --no-push --no-git-tag-version --exact",
-    "all:publish": "lerna exec --no-private -- npm publish --tag latest --access public"
+    "all:publish": "lerna exec --no-private -- npm publish --tag latest --access public",
+    "apps:ocean": "nest start ocean-api"
```
Comment (fuxingloh) [inline package.json:27]: should not have start function within root package, if we need to we can use `npm run start -workspace=ocean-api`.

File: package.json [hunk 2]
```diff
@@ -23,7 +23,12 @@
     "all:clean": "rm -rf ./packages/**/dist && rm -rf ./apps/**/dist && rm -rf ./packages/**/tsconfig.build.tsbuildinfo",
```
Comment (fuxingloh) [inline package.json:23]: Largely this file should be untouched. Unless it concern all packages and apps.

File: nest-cli.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
+
+{
+  "collection": "@nestjs/schematics",
+  "monorepo": true,
+  "compilerOptions": {
+    "webpack": false
+  },
+  "projects": {
+    "ocean-api": {
+      "type": "application",
```
Comment (fuxingloh) [inline nest-cli.json:10]: hmm, I am wondering if we can forgo `nest-cli.json` and just use the barebone setup.

**Comment by kodemon** [comment]
Thanks for the feedback @fuxingloh , I think with the realignment in thinking this PR does not serve a practical purpose. Will move onto the tickets and solve them from the `main` structure. 👍🏻

**Pull request closed without merging**